client/app/collections/filtered.coffee

```diff
@@ -64,6 +64,16 @@ module.exports = class FilteredCollection
         if query
             res = app.contacts.filter filter @query
 
+            # Reduce to get max score and get a median limit, then excludes all
+            # results $lt it.
+            maxScore = res.reduce (currentMax, model) ->
+                Math.max currentMax, scores.get model
+            , 0
+
+            if maxScore > 5
```

Comment:
`5` seems to be the score since results start to be relevnt enough to enable a median filter. Maybe we should adjust it later…